src/stream/src/common/table/state_table.rs

```diff
@@ -211,6 +211,31 @@ where
     }
 }
 
+fn consistent_old_value_op(row_serde: impl ValueRowSerde) -> OpConsistencyLevel {
+    OpConsistencyLevel::ConsistentOldValue(Arc::new(move |first: &Bytes, second: &Bytes| {
```

Comment:
What about first comparing the bytes, then the deserialized values only if we find differences?